### PR TITLE
.mailmap tidying.

### PR DESCRIPTION
**Pull request by hrhino**
Did this while reviewing #119, for all of you people with 10k computers and email addresses.

```
hhoughton:tsec hhoughton$ git checkout master
Switched to branch 'master'
hhoughton:tsec hhoughton$ git shortlog -nse
   239  jose <cardona.josem@gmail.com>
    68  Robert Soeldner <r.soeldner@gmail.com>
    57  jmcardon <cardona.josem@gmail.com>
    43  Jose C <jmcardon@my.yorku.ca>
    19  Jose C <cardona.josem@gmail.com>
    16  Jose Cardona <cardona.josem@gmail.com>
    13  rsoeldner <r.soeldner@gmail.com>
     6  Harrison Houghton <28076058+hrhino@users.noreply.github.com>
     4  Harrison Houghton <hhoughton@learningobjects.com>
     4  Jarred Ward <jarred@webriots.com>
     2  Gabriel Volpe <volpegabriel@gmail.com>
     2  Christopher Davenport <Chris@ChristopherDavenport.tech>
     1  André Rouél <andre.rouel@gmail.com>
     1  The Gitter Badger <badger@gitter.im>
     1  Bjørn Madsen <bm@lego.com>
     1  Bjørn Madsen <bm@aeons.dk>
     1  jose <jose@debbie-ann.kek>
hhoughton:tsec hhoughton$ git checkout mailmap
Switched to branch 'mailmap'
hhoughton:tsec hhoughton$ git shortlog -nse
   375  Jose Cardona <cardona.josem@gmail.com>
    68  Robert Soeldner <r.soeldner@gmail.com>
    13  rsoeldner <r.soeldner@gmail.com>
    11  Harrison Houghton <hhoughton@learningobjects.com>
     4  Jarred Ward <jarred@webriots.com>
     2  Christopher Davenport <Chris@ChristopherDavenport.tech>
     2  Gabriel Volpe <volpegabriel@gmail.com>
     1  Bjørn Madsen <bm@aeons.dk>
     1  Bjørn Madsen <bm@lego.com>
     1  The Gitter Badger <badger@gitter.im>
     1  André Rouél <andre.rouel@gmail.com>
```

@aeons, if you're around, tell me which is your favorite email and I'll update it. Or do it later if you want.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Jose Cardona <cardona.josem@gmail.com>
+Jose Cardona <cardona.josem@gmail.com> <cardona.josem@gmail.com>
+Jose Cardona <cardona.josem@gmail.com> <jmcardon@my.yorku.ca>
+Jose Cardona <cardona.josem@gmail.com> <jose@debbie-ann.kek>
```
Comment (jmcardon) [inline .mailmap]: LOL. Delet this.

Comment (hrhino) [reply .mailmap]: dude I almost made that your primary....

**Comment by jmcardon** [comment]
I think we're good to merge after word from @aeons

File: .mailmap (hunk 1 above)
Comment (rsoeldner) [reply .mailmap]: :sob:

**Comment by aeons** [comment]
Ah, I got a work email in here as well. The @aeons.dk one is my private and main email.

**Pull request merged**